main.go

```diff
@@ -43,17 +44,36 @@ func logHTTPError(w http.ResponseWriter, err string, code int) {
 	http.Error(w, err, code)
 }
 
+func validateStreamKey(streamKey string) bool {
+	validKey := regexp.MustCompile(`^[a-zA-Z0-9_\-]+$`)
```

Comment:
This regex is missing 2 characters, both `~` and `.` are valid for HTTP

Reply:
Thank you @wolfcomp, can you double check the new commit and make sure it's fixed?